pyglet/lib.py

```diff
@@ -222,6 +222,11 @@ def find_library(self, path):
                 sys.frozen is True and pyglet.compat_platform == 'darwin'):
             search_path.append(os.path.join(sys._MEIPASS, libname))
 
+        # conda support
+        if os.environ.get('CONDA_PREFIX', False):
+            search_path.append(os.path.join(os.environ['CONDA_PREFIX'], 'lib',
```

Comment:
If you prefer, we allow a line length of 100 columns in pyglet.

Reply:
Thanks, I've changed this in e0ca014